### DISCUSSION TITLE
best way to call the db

### DISCUSSION THREAD

**Discussion by ubresponsive** [gladly-team/next-firebase-auth#82] [Q&A]
I realise this is an auth module but curious on the best way to call the firestore dB without reinstantiating firebase,
for example if I include below, I get FirebaseError: Firebase: Firebase App named '[DEFAULT]' already exists (app/duplicate-app).

const firebaseApp=firebase.initializeApp(firebaseConfig);
const db=firebase.firestore();
export default db;

**Comment by kmjennison** [comment]
You should be able to initialize Firebase yourself before initializing `next-firebase-auth`. See [this comment](https://github.com/gladly-team/next-firebase-auth/discussions/61#discussioncomment-323977) for an example.

I'm also open to suggestions on how we can make this easier, while keeping in mind I'd like this package to stay fairly focused on auth.
